### PR TITLE
BaSP-MR-44: Create SuperAdmins tests for put/delete

### PR DESCRIPTION
**Pull request by ghost**
Create SuperAdmins tests for put/delete

### CODE REVIEW

File: src/tests/member.test.js
Comment (josedaniele) [inline src/tests/member.test.js:1]: Why did you do the tests for this file in the superadmin PR?

**Review by josedaniele — CHANGES_REQUESTED** [review]

File: src/tests/member.test.js
Comment (danielninfi) [reply src/tests/member.test.js:1]: maybe it happened because I started working after the member merge.

Comment (danielninfi) [reply src/tests/member.test.js:1]: it shouldn't bother at merge time, but yes, it's my mistake.

**Review by OscarLucero — CHANGES_REQUESTED** [review]
You have to add GetAll (Empty)

**Review by OscarLucero — APPROVED** [review]

**Review by josedaniele — APPROVED** [review]

File: src/tests/super-admins.test.js [hunk 1]
```diff
@@ -1,5 +1,162 @@
...
+import request from 'supertest';
+import app from '../app';
+import SuperAdmin from '../models/SuperAdmin';
+import superAdminSeed from '../seeds/super-admins';
+
+beforeAll(async () => {
+  await SuperAdmin.collection.insertMany(superAdminSeed);
+});
+
+describe('PUT /api/super-admin/:id', () => {
```
Comment (JuanMornik) [inline src/tests/super-admins.test.js:10]: Missing status 500 test

File: src/tests/super-admins.test.js [hunk 2]
```diff
@@ -1,5 +1,162 @@
...
+      email: 'probando2@gmail.com',
+      password: 'Password124',
+    };
+    const response = await request(app).put(`/api/super-admin/${id}`).send(data);
+    expect(response.status).toBe(400);
+    expect(response.error).toBeTruthy();
+  });
+});
+
+describe('GET BY ID /api/super-admin/:id', () => {
```
Comment (JuanMornik) [inline src/tests/super-admins.test.js:52]: Missing invalid ID test

File: src/tests/super-admins.test.js [hunk 3]
```diff
@@ -1,5 +1,162 @@
...
+      throw new Error('Error finding SuperAdmin by ID');
+    });
+    const id = '64667748fc13ae7f027631cc';
+    const response = await request(app).get(`/api/super-admin/${id}`).send();
+    expect(response.status).toBe(500);
+    expect(response.error).toBeTruthy();
+  });
+});
+
+describe('DELETE /api/super-admin/:id', () => {
```
Comment (JuanMornik) [inline src/tests/super-admins.test.js:82]: Missing invalid ID test

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**